tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCollationStringByteConverter.java

```diff
@@ -335,7 +335,7 @@ private static String show(String test, ParsePosition parsePosition) {
     }
 
     public static void check() throws Exception {
-        final RuleBasedCollator col = (RuleBasedCollator) Collator.getInstance(ULocale.ENGLISH);
+        final RuleBasedCollator col = (RuleBasedCollator) Collator.getInstance(ULocale.ROOT);
         col.setStrength(Collator.PRIMARY);
         col.setAlternateHandlingShifted(true);
```

Comment:
Could add to CollatorHelper, though maybe unique: strength PRIMARY and  setAlternateHandlingShifted(true)

Reply:
Agreed. Almost any frozen collator could be centralized.